### ISSUE TITLE
F1 (Variation) - "FAQ by category" heading should be H1 not H2

### ISSUE THREAD

**Issue by jszasz** [brharp/hjckrrh#122]
eg. https://aoda.web.uoguelph.ca/ccs/faq/category/76

"Business Phones" Should be an H1 not a H2

**Comment by jszasz** [comment]
Fixed
